modules/jupyter-widget/package.json

```diff
@@ -28,6 +28,9 @@
     "prepublishOnly": "webpack"
   },
   "dependencies": {
-    "@jupyter-widgets/base": "^1.1.10"
+    "@jupyter-widgets/base": "^1.1.10 || ^2"
```

Comment:
`>=1.1.10`?

Reply:
Could try it, was copying a solution to an error described [here](https://github.com/jupyterlab/jupyterlab/issues/7063#issuecomment-523937078).